README.md

```diff
@@ -45,25 +45,25 @@ variables have been set correctly, you can print them in the terminal -- for exa
 
 All the settings the dashboard looks at are in the sample file `sample.env`. This file isn't used by the dashboard, it just
 lists the environment variables that you can copy in your `rc` files:
-  - `TTC_BOTS` are the 3 twitter bots to check, comma separated. The first entry
+  - `TTC_BOTS` -- the 3 twitter bots to check, comma separated. The first entry
```

Comment:
The problem with this is that this changes makes it realy difficult to review your PR.
Pull request should ideally only taggle on feature or issue and should not make any changes to code that is not part of this change-set.

I like the new style. could you create a new PR that "fixes" the README please!?